source/features/index.tsx

```diff
@@ -52,12 +52,12 @@ interface InternalRunConfig {
 
 let log: typeof console.log;
 
-function logError(id: FeatureID, error: Error | string, ...extras: unknown[]): void {
+function logError(id: FeatureID, error: Error | string | unknown, ...extras: unknown[]): void {
 	if (error instanceof TypeError && error.message === 'Object(...)(...) is null') {
 		error.message = 'The element wasn’t found, the selector needs to be updated.';
 	}
 
-	const message = typeof error === 'string' ? error : error.message;
+	const message = typeof error === 'string' ? error : (error as Error).message;
```

Comment:
Avoid assertions when possible. The line already has a typecheck.
```suggestion
	const message = error instanceof Error ? error.message : error; 
```